packages/react-components/src/__tests__/utils.test.tsx

```diff
@@ -73,3 +78,38 @@ describe('getIconFromUrl', () => {
     expect(getIconFromUrl('not a url')).toBeUndefined();
   });
 });
+
+describe('isHashLink', () => {
+  describe.each`
```

Comment:
I think one will suffice :joy: and the description is the same every time, so does not need to be a var :sweat_smile: